src/main/java/core/basesyntax/figure/RightTriangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.figure;
+
+public class RightTriangle extends AbstractFigure implements Figure {
+
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
```

Comment:
Consider removing expressions from the constructor and instead pass pre-calculated variables for better readability. Create variables for 'firstLeg' and 'secondLeg' before passing them to the constructor.